doc/sources/guide/config.rst

```diff
@@ -1,10 +1,22 @@
-.. _configure kivy:
+.. _configuring kivy:
 
-Configure Kivy
+Configuring Kivy
 ==============
 
-The configuration file for kivy is named `config.ini`, and adheres 
-to the `standard INI <http://en.wikipedia.org/wiki/INI_file>`_ format.
+There are three ways to configure Kivy and they can be used in combination.
+
+* Setting :ref:`environment variables` starting with **KIVY_**.  Environment variables can also be set by
+  a program before importing the kivy module.
+* Passing options on the command line.  Importing the kivy module parses command line
+  options, unless KIVY_NO_ARGS is set.  If the option "*--*" is found, parsing stops
+  and remaining options can be parsed the program.  Use *--help* for a list of command
+  line options.
+* Using the kivy configuration file, named `config.ini`.  This adheres
+   to the `standard INI <http://en.wikipedia.org/wiki/INI_file>`_ format.
```

Comment:
This line is over-indented by one space. Line continuations in a list should be indented to the same level as the beginning line (2 spaces in this case, not 3). I don't know how the doc builder will handle this, but at least GitHub's parsing (rich diff) shows that this could cause issues.